### ISSUE TITLE
[website] Tab navigation bar switches positions

### ISSUE THREAD

**Issue by jnnr** [oemof/oemof#56]
On some tabs, the tab navigation bar appears next to the oemof logo. The correct appearance would be beneath the logo.

![oemof-website](https://user-images.githubusercontent.com/32454596/70318305-0b8c8500-1820-11ea-82c9-dd5d71c766ff.png)

**Comment by jnnr** [comment]
Maybe someone with more experience in wordpress than me can help?

**Comment by nesnoj** [comment]
This does only seem to occur in higher resolutions >1400. As I don't know what template/plugins are used I'd look for something where you can configure the responsive design.. (responsive -> layout is adjusted to client's resolution (mobile, tablet, desktop, ...), usually u can define this somewhere).

**Comment by jnnr** [comment]
Thanks for your feedback! It definitely depends on the width of the screen. When there is not enough space the navigation bar gets wrapped to the next line, so it is beneath the logo. The question is how to tell wordpress that it should be always beneath. Something like a linebreak/new line?

I tried to click my way through wordpress and concluding from forums I should look for the "appearance" tab, but cannot find it. Maybe because I do not have the right permissions. Can someone help?

**Comment by busiing** [comment]
I'll ty to fix it. ToDo added to #41.

**Comment by uvchik** [comment]
> On some tabs, the tab navigation bar appears next to the oemof logo. The correct appearance would be beneath the logo.

That's not true. Some time ago we decided, that the menue bar should be next to the log for desktop-screens. For small screens it will be below the logo.

We can decide to change it. We can also decide to use a different template but it is not a bug.

**Comment by jnnr** [comment]
Ok, so this behaviour is intended? I just spoke from intuition and I still think that it makes more sense that the navigation bar starts at the very left on any screen, so there should be a line break before. 

> The reason was, that the headline "Enhancing energy system modelling" is below the centre.

Sorry, I do not understand this. What you mean here?

**Comment by uvchik** [comment]
> I still think that it makes more sense that the navigation bar starts at the very left on any screen, so there should be a line break before.

I do not know what "makes more sense" but I think it is a matter of taste. I like it better as it is, but I do not insist.

> Sorry, I do not understand this. What you mean here?

Sorry, I just wanted to explain why we decided to use it like this. We can change the template only a little bit and so we cannot change the proportion. So we decided that we will win a littel bit more space for content if the menue bar is on the right side. But as I said this is a matter of taste and we can decided it differently now. We also could try to find a better fitting template because there might be some new once available.

![dasf](https://user-images.githubusercontent.com/4760584/78876257-5aa52980-7a4f-11ea-8672-d322ad64a817.png)
